app/models/concerns/journaled/changes.rb

```diff
@@ -57,7 +56,7 @@ def update_columns(attributes, force: false)
   end
 
   class_methods do
-    def journal_changes_to(*attribute_names, as:) # rubocop:disable Naming/UncommunicativeMethodParamName
+    def journal_changes_to(*attribute_names, as:, **opts) # rubocop:disable Naming/UncommunicativeMethodParamName, Metrics/AbcSize
```

Comment:
do we need **opts, or can this be `enqueue_with: {}`? i think that helps to constrain the api a bit

Reply:
Oh, you're right -- lemme follow up with that.